webextensions/api/windows.json

```diff
@@ -791,7 +791,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": true
+                    "version_added": "≤61"
```

Comment:
This was updated in https://github.com/mdn/browser-compat-data/pull/456 so presumably was added earlier